app/src/main/java/com/example/asiantech/playmusic/model/Album.java

```diff
@@ -0,0 +1,14 @@
+package com.example.asiantech.playmusic.model;
+
+import android.graphics.Bitmap;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor(suppressConstructorProperties = true)
+public class Album {
+    private long albumId;
+    private String album;
```

Comment:
WARNING: [AvoidFieldNameMatchingTypeName] It is somewhat confusing to have a field name matching the declaring class name
http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingTypeName